metadata/metadata_test.go

```diff
@@ -335,3 +335,23 @@ func BenchmarkFromOutgoingContext(b *testing.B) {
 		FromOutgoingContext(ctx)
 	}
 }
+
+func BenchmarkFromIncomingContext(b *testing.B) {
+	md := Pairs(
+		"X-My-Header-1", "42",
+	)
```

Comment:
Please change this to be on a single line. Same applies to the next test as well.

Reply:
> Please change this to be on a single line. Same applies to the next test as well.

done